lib/features/popup-menu/PopupMenuItem.js

```diff
@@ -13,25 +13,29 @@ import {
  * @param {boolean} selected
  * @param {function} onMouseEnter
  * @param {function} onMouseLeave
- * @param {function} onClick
+ * @param {function} onSelect
```

Comment:
To me, it feels a bit confusing to talk about `onSelect` outside of select event context. Why don't we call it `onAction` so that it's the same name as in `entry.action`?